### PR TITLE
adding FIPS specific note

### PR DESCRIPTION
**Pull request by PetrMc**
Adding FIPS related language to address paid customers easier.

More details [here](https://tetrate.slack.com/archives/C01PY95JNLB/p1680615157996169)

### CODE REVIEW

File: content/en/download/_index.md [hunk 1]
```diff
@@ -33,11 +33,15 @@ There are additional download of **Istio distributions and istioctl** binaries f
 
 ### Istio Binaries Download
 
-#### Linux Istio Distros
+#### FIPS compliant Distros access
 
-Below you can find direct URL for Istio and istioctl distros: 
+**FIPS compliant distros** are _only_ available to Tetrate Istio Subscription customers. 
+- new customers [order FIPS compliant distribution](/fips-request/) 
+- existing customers - please reach out to Tetrate's support to obtain the required binaries
```
Comment (chirauki) [inline content/en/download/_index.md]:
```suggestion
- new customers [order FIPS compliant distribution](/fips-request/).
- existing customers - please reach out to Tetrate's support to obtain the required binaries.
```

File: content/en/faq/_index.md [hunk 2]
```diff
@@ -59,6 +60,12 @@ all of the builds are available for free. We welcome community participation and
 [community](/community) for updates on new releases, notifications, and regular community events with Istio
 contributors and practitioners.
 
+##### How to download FIPS compliant distro?
+
+FIPS compliant distro is available to active Tetrate Istio Subscription customers. 
+- new customers [order FIPS compliant distribution](/fips-request/) 
+- existing customers - please reach out to Tetrate's support to obtain the required binaries
```
Comment (chirauki) [inline content/en/faq/_index.md]:
```suggestion
- new customers [order FIPS compliant distribution](/fips-request/).
- existing customers - please reach out to Tetrate's support to obtain the required binaries.
```

**Review by ogarrett — APPROVED** [review]

**Review by chirauki — APPROVED** [review]
just a couple of nits, but LGTM

**Comment by ogarrett** [comment]
LGTM, approved

**Comment by PetrMc** [comment]
thank you @chirauki  @ogarrett !

**Review by psbrar99 — APPROVED** [review]
LGTM
Thanks @PetrMc

**Pull request merged**